README.md

```diff
@@ -1 +1,94 @@
-# selfserviceoverlay
\ No newline at end of file
+# selfserviceoverlay
```

Comment:
I think the name should be reconsidered.

It is OVN and Kubernetes specific, generating NAD/s. It will be useful to reflect that.
If I think about it, there is also no need to mention how it is used (i.e. mentioning self-service), that depends on the permission given to it.

Anyway, here are some suggestions, considering that perhaps multiple CRDs may be requested.
`ovn-netattachdef-adapter`

Reply:
Thanks for the suggestion lets do this on a follow up PR, this PR scope is the api version and group/domain.
Also, the readme changes were not supposed to exist my bad.